README.md

```diff
@@ -110,6 +110,8 @@ $ tests/vendor/bin/codecept run tests/acceptance/users.feature
 
 _If you want to see steps then you can use `--steps` option of codeceptio. Check [full codecept command list here](http://codeception.com/docs/reference/Commands#Run)_
 
+**Note**:You can modify the timeout time by changing the value of **TIMEOUT** constant lower for fast machines and higher for slow computers. The **TIMEOUT** constant located in file tests/acceptance/_bootstrap.php
```

Comment:
remove time (duplicate)

changing -> setting

is located in the following file: